### PR TITLE
update for 2023 naming changes

### PR DESCRIPTION
**Pull request by 0xbrayo**
They changed the Naming from camelCase to using underscores.

### CODE REVIEW

**Comment by yussufbiyik** [comment]
Thank you for letting me know, I'll be doing a little test and merging this request in a few days.

**Comment by 0xbrayo** [comment]
updates?

**Comment by yussufbiyik** [comment]
Sorry for the wait, I had midterms.

After doing a few tests, here is the lowest execution time from both your upgraded version and my old one:
Before:
```
Going through StreamingHistory0.json
Going through StreamingHistory1.json
Going through StreamingHistory2.json
Going through StreamingHistory3.json
Going through StreamingHistory4.json
Going through StreamingHistory5.json
Going through StreamingHistory6.json
Going through StreamingHistory7.json
Your total listening time is 1451.76 hours which is 60.49 days!
Execution time: 0.2297048568725586
```
After:
```
Going through StreamingHistory0.json
Going through StreamingHistory1.json
Going through StreamingHistory2.json
Going through StreamingHistory3.json
Going through StreamingHistory4.json
Going through StreamingHistory5.json
Going through StreamingHistory6.json
Going through StreamingHistory7.json
Your total listening time is 1451.76 hours which is 60.49 days!
Execution time: 0.19525432586669922
```

Yours is clearly superior so thank you for the improvement. 
But the problem is my Spotify data does not seem to have the new naming, it's still the old camel case version and script works perfectly. So that's a bit weird but I assume it's just on my end since there was another PR about this a month ago pointing out the same problem as yours.

Long story short, your PR will be merged in a minute, thank you so much for contributing!
